lib/queries/getById.js

```diff
@@ -10,17 +10,13 @@ module.exports = function (challengeId) {
     'challenges.org_id AS org_id,',
     'challenges.active AS active,',
     'organisations.name AS org_name,',
-    'tags.id AS tags_id,',
-    'tags.name AS tags_name',
+    'tags_challenges.tags_id AS tags_id',
```

Comment:
Do you also need to return locations_challenges.locations_id? If yes you will need to do another ```LEFT OUTER JOIN locations_challenges ON (locations_challenges.challenges_id = challenges.id)``` Or do you want to add the locations later on?